pages/linux/named.md

```diff
@@ -0,0 +1,24 @@
+# named
+
+> Execute the DNS (Dynamic Name Service) server daemon and convert host names to IP addresses and vice versa.
```

Comment:
Perhaps this will read better?
```suggestion
> Execute the DNS (Dynamic Name Service) server daemon that converts host names to IP addresses and vice versa.
```